src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
```

Comment:
Let's create more meaningful name for this variable:
```suggestion
  const totalCost = days * 40;
```